src/DTOs/PdfPage.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Spatie\PdfToImage\DTOs;
+
+use Spatie\PdfToImage\Enums\OutputFormat;
+
+class PdfPage
+{
+    public function __construct(
```

Comment:
A protected constructor for DTOs or VOs makes it easier to refactor or extend in the future.

```suggestion
    protected function __construct(
```

Reply:
@Nielsvanpach Could you elaborate on this and/or is it Spatie practice to make DTO constructors protected? I'm definitely leaning towards agreement, but if you could expand a bit I'd appreciate it :+1: